app/Commands/Know/Context.php

```
@@ -0,0 +1,221 @@
+                'blocked' => '🚫',
+                default => '📝'
+            };
+            $details[] = "{$statusIcon} {$status}";
+        }
+
+        if (! empty($details)) {
+            $this->line('   '.implode(' | ', $details));
+        }
+    }
+
+    private function displayContextInfo(KnowledgeService $knowledgeService): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$knowledgeService'. (Unused Code Rules)

(UnusedFormalParameter)